torch/distributions/multivariate_normal.py

```diff
@@ -64,6 +64,17 @@ def _batch_inverse(bmat):
     return flat_inv_bmat.view(bmat.shape)
 
 
+def _batch_mm(bmat1, bmat2):
```

Comment:
Can’t `torch.matmul` handle that?

Reply:
`torch.matmul` is slower in my opinion.

Reply:
@apaszke I have fixed this in the latest commit.